persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_9/Ballerina.toml

```diff
@@ -1,16 +1,12 @@
 [package]
 org = "foo"
-name = "perist_generate_9"
+name = "persist_generate_9"
 version = "0.1.0"
 distribution = "2201.3.0"
 
 [build-options]
 observabilityIncluded = false
 
-[persist.entities]
-provider = "mysql"
-host = ""
-port = 3307
-user = ""
-password = ""
-# database = "entities"
+[persist]
+datastore = "mysql"
+module = "persist_generate_9.entities"
```

Comment:
```suggestion
module = "persist_generate_9.entities"

```

Reply:
fixed in [57989e5](https://github.com/ballerina-platform/persist-tools/pull/148/commits/57989e5ba5b7dbb41b71157fc39b23ae97882031)